### ISSUE TITLE
SideNavigation: SideNavigationSubItem count is not read

### ISSUE THREAD

**Issue by PriyaPowar** [SAP/ui5-webcomponents#5052]
### **Bug Description**
The item count information is not read when the SideNavigation ‘Tree items’ are accessed’. Only the name of the item is read when accessed. Raised as a part of accessibility testing.

### **Expected Behavior**
The item name and the count information also should be read when the tree items are accessed.

### **Steps to Reproduce**
Use accessibility testing software to check if the information is read correctly. SideNavigation react ui5 wiki can be used for testing https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-sidenavigation--default-story.

### **Context**
- UI5 Web Components version:"@ui5/webcomponents": "1.0.0-rc.14"
- OS/Platform: windows
- Browser: chrome
- Affected component: side navigation

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

**Comment by stbodurov** [comment]
Hi @SAP/ui5-webcomponents-topic-rd,

Could you take a look at this feature request? Seems related to another one - #5034

Regards,
Stanislav

**Comment by dimovpetar** [comment]
Hello @PriyaPowar ,
I was able to reproduce the issue only when accessing SideNavigationItem, not SideNavigationSubitem. I tested on this page https://sap.github.io/ui5-webcomponents/playground/components/SideNavigation/ with JAWS 2022. For example "4 of 4" is not read out for the "Events" item
![image](https://user-images.githubusercontent.com/32839090/164009607-50664a6c-2d9c-4b25-83e8-b845c4b2110b.png)
Can you check if the issue is the same?


Best regards,
Petar

**Comment by PriyaPowar** [comment]
Hi [dimovpetar](https://github.com/dimovpetar) ,
Yes, the issue is at SideNavigationItem level. It should specify the count of items under it. Please check.

**Comment by LidiyaGeorgieva** [comment]
Hello @PriyaPowar, the Side Navigation works as expected. When the focus is on some expanded tree item the count of the sub items don't need to be announced. You can see the same behavior on the ARIA best practice example: https://w3c.github.io/aria-practices/examples/treeview/treeview-navigation.html.

**Comment by samanr** [comment]
@LidiyaGeorgieva Is it possible to find out which version would contain this fix?

**Comment by samanr** [comment]
> Hi @samanr,
> 
> There is no issue here. The reading is as expected.
> 
> "the Side Navigation works as expected. When the focus is on some expanded tree item the count of the sub items don't need to be announced"
> 
> Best, Teo

@TeodorTaushanov  We are using MacOS screenreader and it reads as "Treeitem XXX. XXX, group"

We have not tested it using JAWS latest yet. As you can see it does not read out the side navigation position, i.e. i of 2 or 2 of 2. In my case the subitem is rendered using a list.

However my request was to add an accessibility props to the sidenavigation item so we can pass custom accessibility text. I believe it will make it consistent among other screen readers.

**Comment by TeodorTaushanov** [comment]
I'm reopening the ticket for further investigation.

**Comment by TeodorTaushanov** [comment]
Hi,

We improved the accessibility of the ui5-side-navigation in the main branch. 

Note that the different screen readers might read the control differently. For example, JAWS reads "1 of n" only the first time when enter a new tree level. After that when the arrow keys are used, the "2 of n" is not read. This is all up the to screen readers. 


Best,
Teo
